frontend/src/models/admin.js

```diff
@@ -216,6 +216,26 @@ const Admin = {
         return { success: false, error: e.message };
       });
   },
+  setWelcomeMessages: async function (messages) {
+    return fetch(`${API_BASE}/system/set-welcome-messages`, {
+      method: "POST",
+      headers: {
+        ...baseHeaders(),
+        "Content-Type": "application/json",
```

Comment:
Do we need to specify the `application/json` here? All other calls are working without this?